### PR TITLE
[AllBundles] Feature/1047 - Image page part direct upload without media selector

### PR DESCRIPTION
**Pull request by delboy1978uk**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | https://github.com/Kunstmaan/KunstmaanBundlesCMS/issues/1047

### CODE REVIEW

File: src/Kunstmaan/MediaBundle/Controller/ChooserController.php [hunk 1]
```diff
@@ -168,4 +169,130 @@ private function createTypeFormView(MediaManager $mediaManager, $type)
...
+            if (!empty($cKEditorFuncNum)) {
+                $params['CKEditorFuncNum'] = $cKEditorFuncNum;
+                $routeName                 = 'KunstmaanNodeBundle_ckselecturl';
+            } else {
+                $routeName = 'KunstmaanNodeBundle_selecturl';
+            }
+            $linkChooserLink = $this->generateUrl($routeName, $params);
+        }
+
+        $viewVariabels = array(
```
Comment (deZinc) [inline src/Kunstmaan/MediaBundle/Controller/ChooserController.php]: array notation

**Comment by delboy1978uk** [comment]
done!

**Comment by delboy1978uk** [comment]
I tried to resolve this but the javascript file is completely different on both branches, can someone click the resolve conflicts button and have a quick look?

**Comment by delboy1978uk** [comment]
I take it this'll need rebased?

**Pull request closed without merging**